Tests/Integration/IntegrationTests.swift

```diff
@@ -35,7 +35,7 @@ class IntegrationSpec: QuickSpec {
                             return
                         }
 
-                        SignatureVerifier.verify(signature: bundleSignature,
+                        RSignatureVerifier.verify(signature: bundleSignature,
```

Comment:
Is the alignment now off-by-1 below this line?

Reply:
Yes 😞

Reply:
Fixed